Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -134,6 +136,10 @@ def compareVolumeNodes(volumeNode1, volumeNode2):
             comparison += _("Pixel data mismatch") + "\n"
         return comparison
 
+    def hardenAcquisitionGeometryRegularization(self):
+        settings = qt.QSettings()
+        return (settings.value("DICOM/ScalarVolume/AcquisitionGeometryRegularization", "default") == "hardenTransform")
```

Comment:
This will change the behavior of the module - is this intended?

Reply:
yes it is intended, but the default behavior will not be changed (i.e. create and observe transform, without resampling the volume)

Reply:
see https://github.com/Slicer/Slicer/pull/7362/files#diff-ddb2464152946e0fec0e7fcd1ec6cc4aefcd6b8a9ad3ee82a136564525f0c66fR579-R887

Reply:
Okay good, I see now.  I do find I almost always want the harden step, but for now I do prefer doing it manually so I know that it's happened.  But I can see this being confusing for most people.